pkg/handler/handler.go

```diff
@@ -277,29 +257,6 @@ func (m *Modifier) addEnvToContainer(container *corev1.Container, tokenFilePath
 	return changed
 }
 
-// parsePodAnnotations parses the pod annotations that can influence mutation:
-// - tokenExpiration. Overrides the given service account annotation/flag-level
-// setting.
-// - containersToSkip. A Pod specific setting since certain containers within a
-// specific pod might need to be opted-out of mutation
-func (m *Modifier) parsePodAnnotations(pod *corev1.Pod, serviceAccountTokenExpiration int64) (int64, map[string]bool) {
-	// override serviceaccount annotation/flag token expiration with pod
-	// annotation if present
-	tokenExpiration := serviceAccountTokenExpiration
-	expirationKey := m.AnnotationDomain + "/" + pkg.TokenExpirationAnnotation
-	if expirationStr, ok := pod.Annotations[expirationKey]; ok {
-		if expiration, err := strconv.ParseInt(expirationStr, 10, 64); err != nil {
-			klog.V(4).Infof("Found invalid value for token expiration, using %d seconds as default: %v", serviceAccountTokenExpiration, err)
-		} else {
-			tokenExpiration = pkg.ValidateMinTokenExpiration(expiration)
-		}
-	}
-
-	containersToSkip := getContainersToSkip(m.AnnotationDomain, pod)
-
-	return tokenExpiration, containersToSkip
-}
-
```

Comment:
This function has been moved to `pkg/annotations/parser.go` (with some minor changes)